paper-ieee-trans.tex

```diff
@@ -487,13 +486,13 @@ \subsubsection{Quality metrics}
 \textit{Development metrics} are obtained programatically from several sources, namely GitHub
 API and Jenkins service, and graphically displayed as GitHub pages using GrimoireLab
 framework \cite{grimoirelab}. Per-component weekly reports, including the SQA requirement
-completeness, are issued and individually discussed through the different communication channels
+fulfilment, are issued and individually discussed through the different communication channels
 with the development teams. Issue tracking metrics and KPIs are an essential piece of information
 of both feature addition and defect solving, useful to detect and fix misbehaviours while in the
 development phase.
 
 \textit{Release metric} sources are the online repository servers, namely the
-Linux package \cite{indigo-pkg-repo} and DockerHub \cite{indigo-dockerhub} repositories. Jenkins
+Linux packages \cite{indigo-pkg-repo} and DockerHub \cite{indigo-dockerhub} repositories. Jenkins
```

Comment:
should be singular since it refers to repositories -> linux package repositories

Reply:
agree with Pablo

Reply:
ok